_config.yml

```diff
@@ -18,7 +18,7 @@ theme_settings:
   header_text:
   header_text_feature_image:
   footer_text: >
```

Comment:
It looks like this variable (`footer_text`) is no longer being used? If that's true, let's just delete it along with any other unused configs.